gusto/io.py

```diff
@@ -630,6 +630,10 @@ def pick_up_from_checkpoint(self, state_fields):
             self.output.checkpoint_pickup_filename = None
         else:
             raise ValueError("Must set checkpoint True if picking up")
+        # Prevent any steady-state diagnostics overwriting their original fields
```

Comment:
Very picky, but it would be good to have a blank line above this comment to help illustrate that the next segment of code is doing something separate from the above bits of code

Reply:
So ive addressed the points regarding the example and some of the comments but was a bit uncertain on my explaining comments so they will be worth checking.

I think including perturbation in this will be a good idea too so will start having a dig around in that now